src/main/java/com/socialmedia/model/ApplicationUser.java

```diff
@@ -48,6 +48,9 @@
   @MayAcceptNull
   @Column(name = "gender")
   private Gender gender;
+  @MayAcceptNull
```

Comment:
I don't think this annotation is needed here. It means that if you have some incoming Dto with lastActivityTime null from client side, and you map it to entity, it will copy null value instead of skipping the field